src/Kunstmaan/AdminBundle/Repository/ExceptionRepository.php

```diff
@@ -26,8 +27,10 @@ public function markAllAsResolved()
     {
         return $this->createQueryBuilder('e')
             ->update()
-            ->set('e.isResolved', 1)
-            ->where('e.isResolved = 0')
+            ->set('e.isResolved', ':isResolvedTrue')
+            ->where('e.isResolved = :isResolvedFalse')
+            ->setParameter('isResolvedTrue', true, \PDO::PARAM_BOOL)
```

Comment:
I don't know if it's needed to pass the 3th parameter here (I know it brings a speed improvement for `\DateTime` objects), so we should decide to add or not add it. Because now it's a mix of both, so if it's not required I would suggest not adding it.

EDIT: or is it here required because it's a parameter for an update field?

Reply:
Your right, this is not consistent, have removed them

Reply:
parameter was removed